content/git/concepts/branch/branch.md

```diff
@@ -36,6 +36,18 @@ git branch -d <branch-name>
 
 As long as the branch has no uncommitted code, this command will remove the branch entirely. It will no longer be available to check out, and the changed code will be unrecoverable.
 
+Sometimes Git will refuse to delete a branch because it has changes that have not been merged into the upstream or remote version of that branch. In these cases, Git can be _forced_ to delete the branch by changing the flag to a capital "D" or adding the `--force` flag like so:
```

Comment:
```suggestion
Sometimes, Git will refuse to delete a branch because it has changes that have not yet been merged into an existing upstream or remote version of that branch. In such cases, the branch can be forcefully deleted with either the `-d --force` or capital `-D` flags, like so: 
```

Our colleagues in Product prefer to be as minimal with emphasis as possible, which includes leaving out italics in almost all cases.